<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>winAiPhoto</class>
 <widget class="QWidget" name="winAiPhoto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1199</width>
    <height>821</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>0</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>AiPhoto</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Source Photo</string>
   </property>
  </widget>
  <widget class="QLabel" name="LbOutputImage">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>30</y>
     <width>541</width>
     <height>421</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="midLineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string>output image</string>
   </property>
  </widget>
  <widget class="QLabel" name="LbInputImage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>561</width>
     <height>421</height>
    </rect>
   </property>
   <property name="text">
    <string>input image</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PbNext">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>650</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PbResetAll">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>570</y>
     <width>100</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>ResetAll</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PbLast">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>590</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Last</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PbWinBeatify">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>660</y>
     <width>100</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>WinBeatify</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PbSaveFile">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Save File</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>770</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Hint:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PbOpenFile">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="TbHint">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>770</y>
     <width>1101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="PbBackToMain">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>710</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>BackToMain</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PbConfirm">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>470</y>
     <width>100</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
